debugging/book-library/index.html

```diff
@@ -65,7 +65,7 @@ <h1>Library</h1>
           type="submit"
           value="Submit"
           class="btn btn-primary"
-          onclick="submit();"
+          data-atr = "submit"
```

Comment:
Oh, btw, I generally advise data attributes, so perhaps because I've said so! 😉 

With data attributes you pass JSON, you can grab the entire dataset, and you can group like classes but not get in the way of CSS https://developer.mozilla.org/en-US/docs/Learn_web_development/Howto/Solve_HTML_problems/Use_data_attributes

Reply:
Thank you, it was easy to use as well